src/Components/NotificationListItem.js

```diff
@@ -19,27 +20,61 @@ import useMediaQuery from "@mui/material/useMediaQuery";
 // @param {Object} item - The notification object.
 
 export function NotificationListItem({ item, handleClose, index }) {
+  const [localUser, setLocalUser] = useContext(LocalUserContext);
   const theme = useTheme();
+  const location = useLocation();
   const navigate = useNavigate();
   const isMobileWidth = useMediaQuery(theme.breakpoints.down("sm"));
-  const [showGhosts, setShowGhosts] = useState(false);
+
+  // Determine what type of notification this is, in order to select text output
+  let notificationType;
+  if (!item.docType) notificationType = "listComment";
+  if (item.docType === "comments") notificationType = "emojiOnComment";
+  if (item.docType === "reviews") notificationType = "emojiOnReview";
+  if (item.docType === "list") notificationType = "emojiOnList";
+
+  let ignoreCall = notificationType === "emojiOnReview" ? false : true;
```

Comment:
nit: probably could be cleaner without the ternary.  the result of a boolean comparison is already a bool.  something like `const ignoreCall = notificationType !== "emojiOnReview"`